sdk/resourcemanager/azure-resourcemanager-containerservice/src/main/java/com/azure/resourcemanager/containerservice/models/KubernetesClusterAgentPool.java

```diff
@@ -179,6 +179,21 @@
             WithAttach<ParentT> withVirtualNetwork(String virtualNetworkId, String subnetName);
         }
 
+        /**
+         * The stage of a container service agent pool definition allowing to specify the agent pool mode.
+         *
+         * @param <ParentT> the stage of the container service definition to return to after attaching this definition
+         */
+        interface withAgentPoolMode<ParentT> {
+            /**
+             * Specifies the agent pool mode for the agents.
```

Comment:
Prefer to specify that AKS need a system mode agent pool.

Reply:
Change to agent pool default use system mode.